contentScript.js

```diff
@@ -118,14 +118,14 @@
       itemList.appendChild(item);
       item.toolbar = toolbar;
       item.answer = answers[i].description;
-      item.addEventListener('click', insertAnswer);
+      $(item).on('selectmenu:selected', insertAnswer);
```

Comment:
Damn. Are there restrictions on `CustomEvent` `dispatchEvent` between extension sandboxes?

Like if I change `$(item).trigger('selectmenu:selected')` in our code to `item.dispatchEvent(new CustomEvent('selectmenu:selected'))` would that help?